drivers/SmartThings/matter-lock/profiles/lock-without-codes-batteryLevel.yml

```diff
@@ -0,0 +1,23 @@
+name: lock-without-codes-batteryLevel
+components:
+- id: main
+  capabilities:
+  - id: lock
+    version: 1
+    config:
+      values:
+      - key: "lock.value"
+        enabledValues:
+        - locked
+        - unlocked
+        - not fully locked
+  - id: batteryLevel
```

Comment:
I think `batteryLevel` should be last to be consistent with it's position in the other profiles.

Reply:
Alongside this, we should add the `batteryLevel` to the capabilities ordering that Tom put together here so that we can establish a convention: https://smartthings.atlassian.net/wiki/spaces/DEV/pages/618631863/Device+requirements

Reply:
done on both fronts :+1: